src/main/java/space/obminyashka/items_exchange/security/jwt/refresh/RefreshTokenRepository.java

```diff
@@ -0,0 +1,14 @@
+package space.obminyashka.items_exchange.security.jwt.refresh;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.Optional;
+
+public interface RefreshTokenRepository extends JpaRepository<RefreshToken, Long> {
+
+    Optional<RefreshToken> findById(long id);
```

Comment:
I think that method has to be a part of JpaRepository, could you check?

Reply:
Yes, you are right. It will be removed